docs/src/main/asciidoc/spring-data-rest.adoc

```diff
@@ -0,0 +1,441 @@
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `spring-data-rest-quickstart` {quickstarts-tree-url}/spring-data-rest-quickstart[directory].
+
+== Creating the Maven project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,bash,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=spring-data-rest-quickstart \
+    -Dextensions="spring-data-rest,resteasy-jackson,quarkus-jdbc-postgresql"
+cd spring-data-rest-quickstart
+----
+
+This command generates a Maven project with a `spring-data-rest` extension.
+
+If you already have your Quarkus project configured, you can add the `spring-data-rest` extension
+to your project by running the following command in your project base directory:
+
+[source,bash]
+----
+./mvnw quarkus:add-extension -Dextensions="spring-data-rest"
+----
+
+This will add the following to your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-spring-data-rest</artifactId>
+</dependency>
+----
+
+For the tests you will also need a `rest-assured` dependency. Add it to the `pom.xml`:
+[source,xml]
+----
+<dependency>
+    <groupId>io.rest-assured</groupId>
+    <artifactId>rest-assured</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+Note: both `resteasy-jackson` and `resteasy-jsonb` are supported and can be interchanged.
+
+== Define the Entity
+
+Throughout the course of this guide, the following JPA Entity will be used:
+
+[source,java]
+----
+package org.acme.spring.data.rest;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Fruit {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    private String name;
+
+    private String color;
+
+
+    public Fruit() {
+    }
+
+    public Fruit(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+}
+----
+
+
+== Configure database access properties
+
+Add the following properties to `application.properties` to configure access to a local PostgreSQL instance.
+
+[source,properties]
+----
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=quarkus_test
+quarkus.datasource.password=quarkus_test
+quarkus.datasource.jdbc.url=jdbc:postgresql:quarkus_test
+quarkus.datasource.jdbc.max-size=8
+quarkus.datasource.jdbc.min-size=2
```

Comment:
Just remove this, it will generate a warning if you don't have the `initial-size` set.